sdcm/tester.py

```diff
@@ -303,6 +303,11 @@ def __init__(self, *args):  # pylint: disable=too-many-statements
                           {'uniqueMember': unique_members_list, 'userPassword': user_password}]
             self.localhost.add_ldap_entry(ip=ldap_address[0], ldap_port=ldap_address[1],
                                           user=ldap_username, password=LDAP_PASSWORD, ldap_entry=ldap_entry)
+        if self.params.get("prepare_saslauthd"):
+            ldap_entry = [f'uid=cassandra,ou=Person,{self.test_ldap_docker.ldap_base_object}',
+                ['uidObject', 'organizationalPerson', 'top'],
```

Comment:
Let's not use "cassandra" user, but use anything else that isn't builtin user?
scylla-qa? (the one used for authroization?)

Reply:
We need both users. explain: https://github.com/scylladb/scylla-cluster-tests/pull/3120#issuecomment-814091142